### PR TITLE
Rb/magikube path issue

### PR DESCRIPTION
**Pull request by RohitBhosale204**
- Resolved Magikube installation failing issue.
- Resolved gitHub repo creation destroy error.
- Resolved keycloak failing issue.

### CODE REVIEW

File: src/commands/new/index.ts [hunk 1]
```diff
@@ -106,6 +119,10 @@ Creating a new magikube project named 'sample' in the current directory
     // Get the project name from the command line arguments
     const projectName = args.name;
     const terraform = await TerraformProject.getProject(this);
+    const projectConfig = SystemConfig.getInstance().getConfig();
+    let command: BaseCommand | undefined;
+    console.log(projectConfig);
```
Comment (cloudops-calfus) [inline src/commands/new/index.ts:124]: remove console.log

File: src/core/setup-application.ts [hunk 2]
```diff
@@ -134,61 +137,68 @@ export default class CreateApplication extends BaseProject {
...
-            }
-             for (const file of dotFiles) {
-                await this.createFile(`.${file}`, `../magikube-templates/keycloak-auth-service/${file}.liquid`, `./${projectName}/${appName}`);
-            }
-            for (const file of keyCloakFiles) {
-                await this.createFile(file, `../magikube-templates/keycloak-auth-service/${file}.liquid`, `./${projectName}/${appName}/src/keycloak`);
-            }
-            execSync('npm install', {
-                cwd: `${process.cwd()}/${projectName}/${appName}`,
+        const commands = ['docker-compose build'];
```
Comment (cloudops-calfus) [inline src/core/setup-application.ts:154]: we don't need docker-compose

File: src/commands/new/index.ts (hunk 1 above)
Comment (RohitBhosale204) [reply src/commands/new/index.ts:124]: Console removed.

File: src/core/setup-application.ts (hunk 2 above)
Comment (RohitBhosale204) [reply src/core/setup-application.ts:154]: Removed docker compose.

**Review by cloudops-calfus — APPROVED** [review]

**Pull request merged**